chatFunctions.js

```diff
@@ -0,0 +1,97 @@
+const DataStore = require("./datastore.js");
+const url =
+  "mongodb+srv://dfaye:passwordAVDF@cluster0.arzvc.mongodb.net/<dbname>?retryWrites=true&w=majority";
```

Comment:
One of the reasons for using the `.env` file is to remove your password from this line in the code. 

you can delete your password and replace it with your process.env.PASSWORD (or whatever variable you want to use to establish your password.

You'll also need to do an `npm install `dotenv` and inport your `.env` file into this file

The database name could also be replaced because the way it is right now your database name is currently `<dbname>`